src/plugins/vis_default_editor/public/default_editor.tsx

```diff
@@ -109,6 +109,7 @@ function DefaultEditor({
               </EuiResizablePanel>
 
               <EuiResizableButton
+                alignIndicator="start"
```

Comment:
@elastic/kibana-visualizations I noticed while doing a quick grep through Kibana for `<EuiResizableButton>` usages that this button was slightly broken in production due to flex display issues. I fixed this with a height `.scss` change and added this indicator alignment prop to match production.

| Before | After |
|--------|--------|
| ![before](https://github.com/elastic/kibana/assets/549407/263b6eac-68ce-4898-8e50-942016f93dc5) | ![after](https://github.com/elastic/kibana/assets/549407/514206f8-75b7-4ef3-ae9b-2560e70e4fec) |

Reply:
Thanx Cee! I really appreciate it